test/xspec-variable.xspec

```diff
@@ -7,41 +7,174 @@
 <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 
 
-<t:description xmlns:t="http://www.jenitennison.com/xslt/xspec"
-               xmlns:xs="http://www.w3.org/2001/XMLSchema"
-               xmlns:my="http://example.org/ns/my"
-               query="http://example.org/ns/my"
-               query-at="xspec-tested.xql"
-               stylesheet="xspec-tested.xsl">
+<!-- TODO: Make the following changes after other code gets merged onto master branch.
+  - Test that variable/@name should be EQNAME and should be added to EQName test. See
+    test/xspec-eqname.xspec on AirQuick's branch.
```

Comment:
Now this is available in the master. (via #455)

Reply:
Thanks for letting me know! I added EQName tests for XSpec variables as part of https://github.com/xspec/xspec/pull/475/commits/d9840e07d10e5158154a2a76a34a7d6715a1da4c.

Reply:
Thanks for the update including EQName in `@as`. I've never thought of that.

Reply:
No problem!